### DISCUSSION TITLE
吹水区

### DISCUSSION THREAD

**Discussion by github123666** [github123666/cidaren#7] [Announcements]
交流讨论

**Comment by shidao144** [comment]
想问一下，usertoken也是只能使用一次吗？

**Reply by github123666** [reply]
1. code获取的usertoken只能在脚本里面使用 ,如果在其他端访问词达人,则该code也就是usertoken直接失效 。
2. 如果是抓包获取到的则可以一直使用,但是不能重新打开学生端(也就是重新打开一个)可以在没有关闭的webx浏览器里面继续使用

**Reply by github123666** [reply]
不知道说明白没

**Reply by shidao144** [reply]
明白了老表

**Reply by djgao234** [reply]
那请问是更推荐 code 还是 token，那个被风控的概率低一些

**Comment by ahafy** [comment]
弄了好久都不行 一直登不上

**Comment by zhybaba** [comment]
请问可以选择做哪一个单元吗

**Reply by yimeihero** [reply]
同问

**Comment by lihuhuhu** [comment]


**Reply by djgao234** [reply]
同情😭

**Reply by lhfwhahuwq** [reply]
> code获取的usertoken只能在脚本里面使用 ,如果在其他端访问词达人,则该code也就是usertoken直接失效 _

这什么意思，什么叫在脚本里使用？不是填到config.json里面吗？
好兄弟帮帮我

**Comment by anonymity05** [comment]
配置pip install -r requirements.txt后
出现这样Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'

**Reply by lihuhuhu** [reply]
可能路径不对

**Reply by anonymity05** [reply]
怎么搞，下载的是3.12版的

**Reply by MAS-RUI** [reply]
cd命令就好了

**Comment by yimeihero** [comment]
能不能指定哪个任务点，或者是从第几个到第几个

**Reply by yimeihero** [reply]
还有，这个有时长吗大大！

**Reply by yimeihero** [reply]
还有，这个能做快一点吗

**Reply by lihuhuhu** [reply]
> 还有，这个有时长吗大大！
简短说明:
1. 这个账号封了,新的账号我相信聪明的你找的到,，**解决了时长和可视化的问题**
2. 快的问题，应该在其他账号更新
3. 一开始就没打算写指定任务,不是麻烦

**Reply by lihuhuhu** [reply]
> 能不能指定哪个任务点，或者是从第几个到第几个

**一鲸落万物生**   
有个大佬在该代码基础上加了指定任务(我认为可视化这一块没处理好)，我只是个叼毛

**Reply by yimeihero** [reply]
还有这个code和token只用二选一吗

**Reply by yimeihero** [reply]
> > 能不能指定哪个任务点，或者是从第几个到第几个
> 
> **一鲸落万物生** 有个大佬在该代码基础上加了指定任务(我认为可视化这一块没处理好)，我只是个叼毛

谢谢大佬，请问code和token只用二选一吗

**Reply by yimeihero** [reply]
> > 还有，这个有时长吗大大！
> > 简短说明:
> 
> 1. 这个账号封了,新的账号我相信聪明的你找的到,，**解决了时长和可视化的问题**
> 2. 快的问题，应该在其他账号更新
> 3. 一开始就没打算写指定任务,不是麻烦

哪个账号呀，现在这个吗

**Reply by lihuhuhu** [reply]
code是钱 ，token是票，code可以购买token又或者你是内部人员(文档中说了token的获取方式)可以直接拿到票(token),不知道说明白没,两者取一个就行。

**Reply by AkimioJR** [reply]
> code是钱 ，token是票，code可以购买token又或者你是内部人员(文档中说了token的获取方式)可以直接拿到票(token),不知道说明白没,两者取一个就行。

请问token是在`https://app.vocabgo.com/student/`这里找吗，我看了一眼响应体和响应头都没找到

**Reply by lihuhuhu** [reply]
> > code是钱 ，token是票，code可以购买token又或者你是内部人员(文档中说了token的获取方式)可以直接拿到票(token),不知道说明白没,两者取一个就行。
> 
> 请问token是在`https://app.vocabgo.com/student/`这里找吗，我看了一眼响应体和响应头都没找到
单纯获取token, 请求头中好找一点。请求头中叫usertoken我简化了一下叫token。响应体在Auth/Wechat/LoginByWechatCode这个请求里面，代码写的有点乱。找不到可能是你获取工具可能没配置正确

**Comment by hbwnag** [comment]
releases在cmd运行exe出现NameError: name 'exit' is not defined怎么办呢

**Reply by lihuhuhu** [reply]
> releases在cmd运行exe出现NameError: name 'exit' is not defined怎么办呢

这个是因为你其他地方报错引起的，可以截一下报错原因

**Reply by hbwnag** [reply]
<img width="867" alt="屏幕截图 2024-09-19 131440" src="https://github.com/user-attachments/assets/5edd1229-e445-4265-8fc6-23833e33a5ae">
目前是这样

**Reply by hbwnag** [reply]
上面获取题目也是直接走的跳过没有直接答题

**Reply by lihuhuhu** [reply]
> 上面获取题目也是直接走的跳过没有直接答题

应该是作者没考虑周到，你可以尝试手动过掉该题目。重新执行该程序

**Reply by BruceEdc** [reply]
我的是这种
[
<img width="837" alt="85d0136531b710cdf3e92d728bb9b74" src="https://github.com/user-attachments/assets/be6d2bf5-0906-4802-b9b9-c8906b29e426" />
](url)

**Comment by MAS-RUI** [comment]
配置requirements.txt的时候报错链接github超时？
![Uploading IMG_20240924_141457.jpg…]()

**Comment by cellman14** [comment]
![image](https://github.com/user-attachments/assets/3c646c3a-e736-4a08-a191-e53506b444ca)
为什么我安装不了spacy

**Comment by ygxygyd** [comment]
![IMG_20241205_184014](https://github.com/user-attachments/assets/addbd640-8200-455f-888e-2e5079c53182)
显示这个是什么情况啊？

**Reply by PapperX** [reply]
我也是这样 怎么搞啊

**Comment by TCXH321** [comment]
按照文档说的一直显示code已被使用需要重新抓取

**Comment by BruceEdc** [comment]
请问这种情况是怎么一回事
<img width="837" alt="85d0136531b710cdf3e92d728bb9b74" src="https://github.com/user-attachments/assets/2a3e15dd-385b-4147-9d69-01cc31e1fcd6" />

**Comment by SL-Lamp** [comment]
![Screenshot_2025-02-24-08-23-35-05_84d3000e3f4017145260f7618db1d683](https://github.com/user-attachments/assets/59810393-bc49-47ad-b304-daa2304a80fd)
答了一两题就崩溃退出了

**Comment by lihuhuhu** [comment]
> 交流讨论
我也是挺无语的，啥人都有
![image](https://github.com/user-attachments/assets/ff2ec22b-d6a0-4225-b0d2-2a5fb91441c1)

![image](https://github.com/user-attachments/assets/7ad9c78f-86b8-48bc-a7ab-96ff9cad2c50)

**Comment by UNknown1123-cpu** [comment]
想问这个报错怎么解决
![Uploading 微信图片_20250401221425.png…]()

**Comment by liangsili** [comment]
所以出现这样的报错要怎么解决,看前面几位反馈的也是这个报错,自己抓token和code都试过了,救救孩子
![1745596133254](https://github.com/user-attachments/assets/598fc4bd-6df7-4128-84f4-973ab3259e23)

**Reply by lihuhuhu** [reply]
> 所以出现这样的报错要怎么解决,看前面几位反馈的也是这个报错,自己抓token和code都试过了,救救孩子 ![1745596133254](https://private-user-images.githubusercontent.com/163699280/437519824-598fc4bd-6df7-4128-84f4-973ab3259e23.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDU4ODgxNjAsIm5iZiI6MTc0NTg4Nzg2MCwicGF0aCI6Ii8xNjM2OTkyODAvNDM3NTE5ODI0LTU5OGZjNGJkLTZkZjctNDEyOC04NGY0LTk3M2FiMzI1OWUyMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNDI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDQyOVQwMDUxMDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZDMyZDU5N2Q5OTQzZDgyOTcyNzBiODlhYzQzYjcwMDExMzY3M2E3ZTY1MDQxNDg4NjE1ZWY3MGQ0ZTlkZjAyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.E9HBPO8NkvzXpX_ofMcn3hljnPHuDMWKvaREtKiF21k)

梯子关了或抓包软件关了，这个项目已经停止维护了。大概率用不了

**Comment by PapperX** [comment]
[capture_20251109121301382.bmp](https://github.com/user-attachments/files/23436003/capture_20251109121301382.bmp)
wei'shen为什么会这样啊 拜托怎么搞啊
